packages/components/pages/f-mfa/stories/story.helper.js

```diff
@@ -21,23 +17,23 @@ const apiStates = {
 
 // Mocks for the API calls
 const otpPOST200 = {
-    url: `http://localhost:8080/${SUBMIT_LOGIN_CHALLENGE_URL}`,
+    url: 'http://localhost:8080/mfa/validate',
     method: httpVerbs.post,
     responseStatus: httpStatusCodes.ok,
     requestData: { mfa_token: 'ABC123', otp: 'otp123' }, // eslint-disable-line camelcase
```

Comment:
Super minor (not for this pr), It might be worth updating this disabled line to a more specific eslint disable, i think it's camelCase?

Reply:
If I'm understanding you correctly, we've already got the `camelcase` specifier 😄

Reply:
My bad.